examples/ffmpeg_rtsp_ex.cpp

```diff
@@ -46,13 +48,6 @@ try
         return 0;
     }
 
-    if (!parser.option("i"))
-    {
-        cout << "Missing -i" << endl;
-        parser.print_options();
-        return 0;
-    }
-
```

Comment:
Keep this, since `i` is a required option.  The error message you get otherwise is less clear.

Reply:
Yeah I was wondering if the command line parser could be improved at some later stage. otherwise the error message is a stack trace.

Reply:
Yeah.  There are a bunch of check functions you can call which throw errors that look nice on the terminal.  Use those to validate the CLI.  And we can add more of them for whatever anyone wants.